lti_consumer/lti_xblock.py

```diff
@@ -656,7 +656,11 @@ def lti_provider_key_secret(self):
         """
         for lti_passport in self.course.lti_passports:
             try:
-                lti_id, key, secret = [i.strip() for i in lti_passport.split(':')]
+                # NOTE(sksankarraj)  Now it accepts lti_passports with more than one colons for lit key.
```

Comment:
Thank you @sksankarraj for the fast improvement. Two minor nits:
 - as @nedbat mentioned in a comment above: "git blame" handles authority well so the name in the comment looks odd, sorry for misleading ))
- typo `s/lit key/lti key/`?

Reply:
@idegtiarov Thanks for the review again, sorry for the typo. I have updated the comment with more meaningful content. Please let me know if its okay.